torch/csrc/distributed/rpc/script_functions.cpp

```diff
@@ -12,14 +12,16 @@ namespace distributed {
 namespace rpc {
```

Comment:
what about the file name here? do we need to keep it consistent as `torchscript_functions.h/cpp`?